src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java

```diff
@@ -160,14 +152,17 @@ public void onRender(RenderHUDEvent event) {
 
             GlStateManager.color(1, 1, 1, in < 1000 ? (float) Math.pow(in / 1000D, 2) : fadeOutAlpha);
             GlStateManager.translate(imageWidth, 10, 0);
+
             Gui.drawScaledCustomSizeModalRect(0, 0, 0, 0, 1200, 675, imageWidth * 2, imageWidth, 1200, 675);
+            System.out.println("victory royale (debug)");
```

Comment:
You've left a debug statement in the code.

Reply:
Outdated branch 🙁 
Removed.